examples/sample_manifest_extended.json

```diff
@@ -0,0 +1,169 @@
+{
+"metadata": {
+    "@type": "http://bib.schema.org/Audiobook",
+    "identifier": "https://librivox.org/flatland-a-romance-of-many-dimensions-by-edwin-abbott-abbott/",
+    "title": "Flatland: A Romance of Many Dimensions",
+    "author": [{
+        "name": {
+            "ru": "Эдвин Эбботт Эбботт",
+            "en": "Edwin Abbott Abbott"
+        },        
+        "sort_as": {
```

Comment:
We carry books in Spanish, Russian, Chinese, etc..  We do not currently have the ability to display the author name based on the user's system language preferences.  Making sure that we keep a place for the versions of author's names to be kept is one of the steps in solving the problem.